manager/notifier.go

```diff
@@ -652,6 +654,7 @@ func (n *notifier) sendOpsGenieNotification(op notificationOp, config *pb.OpsGen
 			return err
 		}
 	case notificationOpResolve:
+		postUrl = *opsgenieAPIURL + "/close"
 		msg := &opsGenieMessageClose{
 			ApiKey: config.GetApiKey(),
```

Comment:
Actually this was that way before, but `ApiKey` should also be renamed to `APIKey`.